### PR TITLE
update azure_armclient log

### PR DESCRIPTION
**Pull request by lovebaby979**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
update azure_armclient log
```

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go [hunk 1]
```diff
@@ -174,7 +174,7 @@ func (c *Client) Send(ctx context.Context, request *http.Request) (*http.Respons
 			regionalErrStr = regionalError.Error().Error()
 		}
 
-		klog.V(5).Infof("Send.sendRegionalRequest failed to get response from regional host, error: '%s'. Ignoring the result.", regionalErrStr)
+		klog.V(5).InfoS("Send.sendRegionalRequest failed to get response from regional host. Ignoring the result.", "error", regionalErrStr)
```
Comment (lala123912) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go:177]: error->err, suggested in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```
With the exception of acronyms like "IP" and the standard "err", don't shorten names. Don't use addr, do use address.
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go [hunk 2]
```diff
@@ -390,15 +390,15 @@ func (c *Client) PutResources(ctx context.Context, resources map[string]interfac
 			response, err := c.WaitForAsyncOperationResult(ctx, future, "armclient.PutResource")
 			if err != nil {
 				if response != nil {
-					klog.V(5).Infof("Received error in WaitForAsyncOperationResult: '%s', response code %d", err.Error(), response.StatusCode)
+					klog.V(5).InfoS("Received error in WaitForAsyncOperationResult", "error", err.Error(),"response code",  response.StatusCode)
 				} else {
-					klog.V(5).Infof("Received error in WaitForAsyncOperationResult: '%s', no response", err.Error())
+					klog.V(5).Infof("Received error in WaitForAsyncOperationResult, no response", "erroor",err.Error())
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go:395]: Typo

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go [hunk 3]
```diff
@@ -426,29 +426,29 @@ func (c *Client) PutResources(ctx context.Context, resources map[string]interfac
...
 	}
 
 	response, err := c.WaitForAsyncOperationResult(ctx, future, "armclient.PutResource")
 	if err != nil {
 		if response != nil {
-			klog.V(5).Infof("Received error in WaitForAsyncOperationResult: '%s', response code %d", err.Error(), response.StatusCode)
+			klog.V(5).InfoS("Received error in WaitForAsyncOperationResult","error", err.Error(), "responseCode",response.StatusCode)
 		} else {
-			klog.V(5).Infof("Received error in WaitForAsyncOperationResult: '%s', no response", err.Error())
+			klog.V(5).InfoS("Received error in WaitForAsyncOperationResult, no response", "erro", err.Error())
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go:445]: Typo

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go [hunk 4]
```diff
@@ -466,29 +466,29 @@ func (c *Client) PatchResource(ctx context.Context, resourceID string, parameter
...
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "patch.send", resourceID, clientErr.Error())
+		klog.V(5).Infos("Received error in patch.send", "resourceID", resourceID, "error", clientErr.Error())
 		return nil, clientErr
 	}
 
 	response, err := c.WaitForAsyncOperationResult(ctx, future, "armclient.PatchResource")
 	if err != nil {
 		if response != nil {
-			klog.V(5).Infof("Received error in WaitForAsyncOperationResult: '%s', response code %d", err.Error(), response.StatusCode)
+			klog.V(5).InfoS("Received error in WaitForAsyncOperationResult", "error", err.Error(), "response code %d", response.StatusCode)
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go:483]:
```suggestion
			klog.V(5).InfoS("Received error in WaitForAsyncOperationResult", "error", err.Error(), "responseCode", response.StatusCode)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go [hunk 5]
```diff
@@ -135,19 +135,19 @@ func (c *Client) Send(ctx context.Context, request *http.Request) (*http.Respons
 	}()
 
 	bodyString := string(bodyBytes)
-	klog.V(5).Infof("Send.sendRequest original error message: %s", bodyString)
+	klog.V(5).InfoS("Send.sendRequest original error message", "responseBody", bodyString)
 
 	// Hack: retry the regional ARM endpoint in case of ARM traffic split and arm resource group replication is too slow
 	var body map[string]interface{}
 	if e := json.Unmarshal(bodyBytes, &body); e != nil {
```
Comment (shubheksha) [inline staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go:142]: If you're up for it, it'd be great to change the `e` to `err` to make it clearer

**Review by shubheksha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**